app/controllers/users_controller.rb

```diff
@@ -2,9 +2,7 @@ class UsersController < ApplicationController
   respond_to :html
   before_action :authenticate_user!
   before_action :find_user, only: [:edit, :show, :update, :destroy]
```

Comment:
I know that you didn't touch this line, but what do you think of this:

`before_action :find_user, except: [:index]`

?